test/performance/alignment/global_affine_alignment_benchmark.cpp

```diff
@@ -175,7 +176,7 @@ void seqan2_affine_dna4_collection(benchmark::State & state)
         seqan::globalAlignmentScore(vec1, vec2, seqan::Score<int>{4, -5, -1, -11});
     }
 
-    state.counters["cells"] = pairwise_cell_updates(views::zip(vec1, vec2), affine_cfg);
+    state.counters["cells"] = pairwise_cell_updates(seqan3::viewszip(vec1, vec2), affine_cfg);
```

Comment:
How does this pass through CI :laughing:

Reply:
Oh this is seqan2 Benchmarks. We really need nightlies for this! @marehr :trollface:?